docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.buttons.scss

```diff
@@ -41,33 +79,137 @@
   }
 
   @include hb-traditional {
-    font-family: $hb-traditional-font--sans;
+    &:active,
+    &:focus {
+      box-shadow: 0 0 0 hb-calculate-rems(1px) hb-get-pairing-color('secondary-active', $hb-traditional-default, $ht-traditional-pairings);
+      box-shadow: 0 0 0 hb-calculate-rems(1px) var(--palette--secondary-active);
+    }
+  }
+}
+
+@mixin hb-big-button {
+  @include hb-button;
+  font-size: hb-calculate-rems(18px);
+  border-width: hb-calculate-rems(3px);
+
+  @include hb-themes(('colorful', 'airy')) {
+    padding: hb-calculate-rems(8px) hb-calculate-rems(20px);
+  }
+
+  @include hb-traditional {
+    padding: hb-calculate-rems(15px) hb-calculate-rems(35px);
+  }
+}
+
+@mixin hb-secondary-button {
+  @include hb-button-base;
+  @include hb-pairing-color('border-color', 'secondary');
+  @include hb-pairing-color('color', 'secondary');
+
+  &:hover {
+    @include hb-global-color('color', 'white');
+    @include hb-pairing-color('background-color', 'secondary');
+    @include hb-pairing-color('border-color', 'secondary');
+  }
+
+  &:active,
+  &:focus {
+    background-color: transparent;
+    @include hb-pairing-color('color', 'secondary');
```

Comment:
I honestly don't remember what the focus state of the secondary button looked like before, so it may have changed at some point from what is shown on the design comps. But I wanted to share that it looks different. Colorful design comps: https://www.figma.com/file/OFuSahKB6uCD4c17vHR9iE/🎨--Colorful-Theme---Team-Library?node-id=0%3A1

If these styles are a reflection of a design chisel, please disregard. 😄 

![colorful focus state](https://user-images.githubusercontent.com/12678977/81188010-ca410280-8f82-11ea-83bf-51575bf0f5be.png)

Reply:
I was going off how the buttons looked on prod but after pairing w/Merani it looks like both the designs and prod were out of date. She updated designs and the focus state should match now.